src/System.Management.Automation/engine/CommandCompletion/CompletionAnalysis.cs

```diff
@@ -564,6 +582,15 @@ internal List<CompletionResult> GetResultHelper(CompletionContext completionCont
                                 // Handle scenarios such as 'gci | Format-Table @{Label=<tab>' if incomplete parsing of the assignment.
                                 return null;
                             }
+                            else if (lastAst is AssignmentStatementAst assignmentAst2)
+                            {
+                                completionContext.ReplacementIndex = replacementIndex += tokenAtCursor.Text.Length;
```

Comment:
I see original code has comments like "// Handle scenarios such as ...". Make sense add such comments for new cases?

Reply:
I'll add some comments